src/01-general/custom-fields.apib

```diff
@@ -0,0 +1,131 @@
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + ids: `74855f4a-2b61-429c-81d8-c79ad3675a76`,`1a68ac98-b41f-4c60-af12-895c43771385` (array[string], optional)
+            + context: `contact` (enum[string], optional) - The context of the custom field definition
+                + Members
+                    + contact
+                    + company
+                    + deal
+                    + project
+                    + milestone
+                    + product
+                    + invoice
+                    + subscription
+                    + ticket
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum[string], required)
+                    + Members
+                        + label
+                + order (enum[string], optional)
+                    + Members
+                        + asc
+            + Default
+                + (object)
+                    + field: `label`
+                    + order: `asc`
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `74855f4a-2b61-429c-81d8-c79ad3675a76` (string)
+                + context: `contact` (enum[string])
+                    + Members
+                        + contact
+                        + company
+                        + deal
+                        + project
+                        + milestone
+                        + product
+                        + invoice
+                        + subscription
+                        + ticket
+                + type (enum[string])
+                    + Members
+                        + single_line - Single line text
+                        + multi_line - Multiline text
+                        + single_select - Single selection - has configuration
+                        + multi_select - Multiple selection - has configuration
+                        + date - Date
+                        + money - Money
+                        + auto_increment - Automatically incrementing field  - has configuration
+                        + integer - Integer
+                        + number - Number
+                        + boolean - Yes/No
+                        + email - Email address
+                        + telephone - Phone number
+                        + url - URL
+                        + company - Company
+                        + contact - Contact
+                        + product - Product
+                        + user - User
+                + label (string)
+                + group (string)
+                + required: false (boolean)
+                + configuration (object)
+                    + One Of
+                        + options: foo, bar (array[string])
+                        + start_value: 1 (number)
```

Comment:
B: The value is saved as the next auto increment value. Better call this: `next_value`
@WouterSioen @mrnnn